cypress/e2e/Courses/Certificates/StandardPlanUnawardedCourseCertificate.feature

```diff
@@ -0,0 +1,26 @@
+Feature: Unawarded Premium Course Certificates for Standard Plan Users
+  Background:
+    Given a standard plan user has some completed unawarded premium courses
+    And the premium courses page has been navigated to
+
+  Scenario: Standard plan users should see the call to action "Review course" for completed and unawarded premium courses
+    Given an iOS mobile device is not being used
```

Comment:
As they are unawarded courses, we don't need to distinguish the device because certificates cannot be downloaded